app/models/application_history.rb

```diff
@@ -12,13 +20,15 @@ def load_events
     RailsEventStore::Projection
       .from_stream("Assigning$#{application.id}")
       .init(-> { [application_submitted_event] })
-      .when(
-        [Assigning::AssignedToUser, Assigning::UnassignedFromSelf],
-        ->(state, event) { state << ApplicationHistoryEvent.from_event(event) }
-      ).run(Rails.application.config.event_store).reverse
+      .when(ApplicationHistory::EVENT_TYPES,
```

Comment:
Can we chat about this

Reply:
Yes, it will possibly be gone in the next pr.